documentation/modules/post/windows/manage/persistence_service.rb

```diff
@@ -0,0 +1,100 @@
+## Overview
+This Module will generate and upload an executable to a remote host and make it persistent service.
+It will create a new service which will start the payload whenever the service is running. Privilege is required.
+
+## Module Options
+LHOST  IP of host that will receive the connection from the payload.
+LPORT  Port for Payload to connect to.
+OPTIONS Comma separated list of additional options for payload if needed in 'opt=val,opt=val' format.
+PAYLOAD The payload to use in the service.
+SESSION The session to run this module on.
+
+RetryTime The retry time that shell connect failed. 5 seconds as default.
+RemoteExePath  The remote victim exe path to run. Use temp directory as default.
+RemoteExeName  The remote victim name. Random string as default.
+ServiceName    The name of service. Random string as default.'
+ServiceDescription The description of service. Random string as default.
+
+
+## Verification steps
+* get session on target
+* `use post/windows/manage/persistence_service`
+* `set payload <payload>`
+* `set lport <lport>`
+* `set lhost <lhost>`
+* `set handler true`
+* `run`
+
+## Usage
+```
```

Comment:
Usually we do a h3 `###` here and put the target version, so maybe 'Windows 7 sp1 x64' so that additional ones could be added later